tool/site/client/tests/pages/validatePage.spec.tsx

```diff
@@ -31,8 +31,8 @@ test("Displays validate result", async () => {
     status: 200,
     response: {
       success: true,
-      request: "JSON Request",
-      response: "JSON Response"
+      request: {req: "JSON Request"},
```

Comment:
why are these changes being made (lines 34-35 & 45-46)?

Reply:
With the previous implementation we basically called `JSON.stringify` on the `request` and `response` fields. This was a little weird but didn't error as `JSON.stringify` can accept a string as a param

The new view component however needs to receive an object so we need this mock to return an `object` rather than simply a `string`